ape_safe/client/mock.py

```diff
@@ -83,7 +88,18 @@ def post_transaction(
                 signatureType=SignatureType.EOA,
             )
             for signer, sig in signatures.items()
+            if signer in owners
         )
+
+        # Ensure that if this is a zero-conf SafeTx, that at least one signature is from a delegate
```

Comment:
Idea: rather than simulate a different service-behavior, we could use an `assert` statement here, since this is a testing class and we are really focused on testing in that sense

Reply:
70d1ee7

Reply:
I rolled this back because it's better that it mimics the behavior of the real client